pkg/cvo/sync_worker.go

```diff
@@ -956,14 +966,14 @@ func (w *SyncWorker) apply(ctx context.Context, work *SyncWork, maxWorkers int,
 				continue
 			}
 			if err := task.Manifest.Include(nil, nil, nil, &capabilities, work.Overrides); err != nil {
-				klog.V(2).Infof("Skipping precreation of %s: %s", task, err)
+				klog.V(manifestVerbosity).Infof("Skipping precreation of %s: %s", task, err)
 				continue
 			}
 			if err := w.builder.Apply(ctx, task.Manifest, payload.PrecreatingPayload); err != nil {
-				klog.V(2).Infof("Unable to precreate resource %s: %v", task, err)
+				klog.V(manifestVerbosity).Infof("Unable to precreate resource %s: %v", task, err)
```

Comment:
I'd rather keep this at two, so we log the first error.  As you have it, we'll presumably soon be in a new sync round with `Attempt > 0`, and we'll log any second precreation error then.  But it still seems like it might be useful for incident timinling to have a timestamp logged for the first round's error.

Reply:
I agree we should not change the logging level for the precreate resource error.